databricks/koala/structures.py

```diff
@@ -534,10 +560,14 @@ def dropna(self, axis=0, how='any', thresh=None, subset=None, inplace=False):
             else:
                 columns = list(self.columns)
 
+            def pred(c):
```

Comment:
Since you are repeating this logic twice already, would you mind going the whole way and defining `PandasLikeSeries.inull`? We can add the tests later when fully implementing #64